src/openrct2-ui/audio/FileAudioSource.cpp

```diff
@@ -202,4 +202,47 @@ namespace OpenRCT2::Audio
         }
         return source;
     }
+
+    IAudioSource* AudioSource::CreateStreamFromWAV(std::unique_ptr<std::istream> stream)
+    {
+        using streamptr = std::unique_ptr<std::istream>*;
+        auto data = new std::unique_ptr<std::istream>(std::move(stream));
```

Comment:
Isn't the `new` here unnecessary? Sounds like you could just move directly

Reply:
See line 214.